Formula/when.rb

```diff
@@ -1,8 +1,8 @@
 class When < Formula
   desc "Tiny personal calendar"
   homepage "http://www.lightandmatter.com/when/when.html"
-  url "https://deb.debian.org/debian/pool/main/w/when/when_1.1.38.orig.tar.gz"
```

Comment:
saw this as well, looks like this is only on the github side yet. Not sure if we want to wait a little bit.

Reply:
`https://repology.org/project/when/versions` repology reference